pkgs/tools/filesystems/cryfs/default.nix

```diff
@@ -25,7 +25,8 @@ stdenv.mkDerivation rec {
       url = "https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-fs/cryfs/files/cryfs-0.10.2-unbundle-libs.patch?id=192ac7421ddd4093125f4997898fb62e8a140a44";
       sha256 = "0hzss5rawcjrh8iqzc40w5yjhxdqya4gbg6dzap70180s50mahzs";
     })
-  ];
+  ] # Backported from https://github.com/cryfs/cryfs/pull/378
+    ++ lib.optional stdenv.isDarwin ./use-macfuse.patch;
```

Comment:
Can we fetch the patch instead?

Reply:
The patch from the linked PR only applies against the `develop` branch, so it had to be backported for the current release.

Reply:
Can we apply the patch unconditional?

Reply:
Fixed.